doc/footer.md

```diff
@@ -0,0 +1,3 @@
+## Contributors
```

Comment:
I think this should be moved to the end of the header file to make it easy to find.

Reply:
In this case, shouldn't we also move it to the end of the header in the other repositories?

Reply:
We can decide on what to do in other repositories when we modify README files there.